tiledb/cf/engines/netcdf4_engine.py

```diff
@@ -514,21 +634,22 @@ def from_group_to_attr_per_array(
                 to the tiles of those dimensions in the generated NetCDF array. The
```

Comment:
In the description of tiles_by_var and tiles_by_dims is the "generated NetCDF array" correct? Or should this be "generated TileDB array"? I still do not completely understand these parameters...